io.fossbr.WhatsAppQT.yml

```diff
@@ -0,0 +1,33 @@
+app-id: io.fossbr.WhatsAppQT
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15'
+command: WhatsAppQT
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+- "--env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess"
+- "--talk-name=org.kde.StatusNotifierWatcher"
+- "--own-name=org.kde.*"
+cleanup-commands:
+- /app/cleanup-BaseApp.sh
+modules:
+- name: WhatsAppQT
+  buildsystem: qmake
+  build-options:
+    env:
+    - QMAKEPATH=/app/lib
+  config-opts:
+  - QMAKE_INCDIR+=/app/include/QtWebEngine
+  - QMAKE_INCDIR+=/app/include/QtWebEngineCore
+  - QMAKE_INCDIR+=/app/include/QtWebEngineWidgets
+  sources:
+  - type: dir
```

Comment:
this will build nothing. you need to specify a tarball or a git repository where to get the source.